include/roaring/portability.h

```diff
@@ -27,11 +27,10 @@
 #include <malloc.h> // this should never be needed but there are some reports that it is needed.
 #endif
 
-#if __SIZEOF_LONG_LONG__ != 8
+#if defined(__SIZEOF_LONG_LONG__) && __SIZEOF_LONG_LONG__ != 8
```

Comment:
@madscientist 

As per your links, the "long long" specification comes from C99. Granted, that's what we expect, but it is not like this was built into the language decades ago. (For some reason, in C, C99 is still considered bleeding edge by some.)

> So, while it might be useful as documentation for this check to be there, there's no architecture existing today that I'm aware of where it will be false

Ok. I am responsible for this check and it was indeed quite paranoid...  but I am not sure it is harmful.